Sources/XcodeProj/Utils/BuildSettingsProvider.swift

```diff
@@ -266,6 +270,10 @@ public class BuildSettingsProvider {
                 "APPLICATION_EXTENSION_API_ONLY": "YES",
                 "LD_RUNPATH_SEARCH_PATHS": ["$(inherited)", "@executable_path/Frameworks", "@loader_path/Frameworks"],
             ]
+        case (.visionOS, .framework):
+            [
+                "LD_RUNPATH_SEARCH_PATHS": ["$(inherited)", "@executable_path/Frameworks", "@loader_path/Frameworks"],
+            ]
         case ([.iOS, .tvOS, .watchOS], .staticLibrary):
```

Comment:
I suspect visionOS is needed in a few other cases such this one (and the app extensions one too)

```suggestion
        case ([.iOS, .tvOS, .watchOS, .visionOS], .staticLibrary):
```